12/homework_12.1.py

```diff
@@ -0,0 +1,90 @@
+import json
+
+my_dict = {
+    'Dima': 1234,
+    'Maks': 2234,
+    'Stas': 3234,
+    'Tim': 4234,
+    'Oleg': 5234
+}
+
+# Перевіримо чи існує файл phonebook.txt, якщо ні, то створимо його із заздалегідь створеною базою абонентів (my_dict)
+try:
+    with open('phonebook.txt') as book:
+        json.load(book)
```

Comment:
General comment for this line and the whole solution: the key point of this task is to use file as a database but at the same time transorm the data to a format that will be comfortable to work with. So, I would suggest you to initialize here the data to the `my_dict` variable (dict) and you can work with this variable as before. For readonly operations (stats, list, show) you can use `my_dict` and don't read from a file. For operations that change the structure of your phonebook you can also use `my_dict` as before, but after adding or deleting some record you should update the file by rewriting it with updated `my_dict` content (update `my_dict` -> open file -> dump `my_dict`'s content to a json string with json.dumps -> rewrite the file with this string)